platform/darwin/src/MGLAttributionInfo.h

```diff
@@ -59,6 +77,19 @@ MGL_EXPORT
  */
 - (nullable NSURL *)feedbackURLAtCenterCoordinate:(CLLocationCoordinate2D)centerCoordinate zoomLevel:(double)zoomLevel;
 
+/**
+ Returns a copy of the current `title` formatted accordingly to `style`.
+ 
+ Example: If `style` = `MGLAttributionInfoStyleShort` and
+ `MGLAttributionInfo.title` = `OpenStreetMap` then it will format
+ `MGLAttributionInfo.title` = `OSM`
```

Comment:
Providing an example is a good idea. I think it would read better as a sentence, though:

> If the `style` property is set to `MGLAttributionInfoStyleShort` and the `title` property is set to `OpenStreetMap`, then this method returns `OSM`.

(jazzy will autolink `title`, since it’s in the same class, so no need to qualify it with `MGLAttributionInfo`.)